packages/wpcom-template-parts/src/universal-header-navigation/index.tsx

```diff
@@ -0,0 +1,447 @@
+/* eslint-disable no-restricted-imports */
+import { WordPressWordmark } from '@automattic/components';
+import { useLocalizeUrl, useIsEnglishLocale } from '@automattic/i18n-utils';
+import { useTranslate, getLocaleSlug } from 'i18n-calypso';
+import { useState } from 'react';
+import { addQueryArgs } from 'calypso/lib/route';
```

Comment:
Sshould use the pacakge version. 
```suggestion
import { addQueryArgs } from '@wordpress/url';
```